### PR TITLE
docs(documentation-website):Add a "What is" to the home page 

### LINKED ISSUE #745: Add a "What is" to the home page
Example:

`@idrinth/api-bench` is an API performance measuring and monitoring framework, that strives to provide the best developer experience possible while keeping it easy to expand and configure.

### PR DESCRIPTION
**Pull request by akhil0203**
# The Pull Request is ready

- [x] fixes #745

### CODE REVIEW

File: framework/README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/Idrinth/api-bench/badge.svg?branch=master)](https://coveralls.io/github/Idrinth/api-bench?branch=master)
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/3171affc728048da8df4fe36b6d4771e)](https://app.codacy.com/gh/Idrinth/api-bench/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Idrinth_api-bench&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Idrinth_api-bench)
+[![Code Factor](https://www.codefactor.io/repository/github/idrinth/api-bench/badge/master)](https://www.codefactor.io/repository/github/idrinth/api-bench/overview/master)
```
Comment (Idrinth) [inline framework/README.md:6]: this shouldn't be in here

File: README.md [hunk 2]
```diff
@@ -3,6 +3,13 @@
 [![Coverage Status](https://coveralls.io/repos/github/Idrinth/api-bench/badge.svg?branch=master)](https://coveralls.io/github/Idrinth/api-bench?branch=master)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Idrinth_api-bench&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Idrinth_api-bench)
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/3171affc728048da8df4fe36b6d4771e)](https://app.codacy.com/gh/Idrinth/api-bench/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
+[![Code Factor](https://www.codefactor.io/repository/github/idrinth/api-bench/badge/master)](https://www.codefactor.io/repository/github/idrinth/api-bench/overview/master)
+
+## What is `idrinth/api-bench` ?
```
Comment (Idrinth) [inline README.md:8]: this is missing on the website

**Review by Idrinth — CHANGES_REQUESTED** [review]

File: framework/README.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/3171affc728048da8df4fe36b6d4771e)](https://app.codacy.com/gh/Idrinth/api-bench/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Idrinth_api-bench&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Idrinth_api-bench)
 
+
```
Comment (Idrinth) [inline framework/README.md]: why an added empty line? Wouldn't the text from the root readme fit in here well?

**Review by Idrinth — CHANGES_REQUESTED** [review]
Still missing the documentation website changes.

File: framework/README.md (hunk 3 above)
Comment (akhil0203) [reply framework/README.md]: hey @Idrinth can you please mention specifically(if possible file route) where to add " what is "

I'm confused with the documentation thank you.

Comment (Idrinth) [reply framework/README.md]: hi @akhil0203 the documentation hompage is in:
```
documentation-website/src/pages/home/index.tsx
```
The text goes into
```
documentation-website/language/en.yml
```

Comment (akhil0203) [reply framework/README.md]: thanks a lot @Idrinth

**Pull request closed without merging**